lib/src/svg/colors.dart

```diff
@@ -36,21 +36,27 @@ Color parseColor(String colorString) {
 
   // handle rgb() colors e.g. rgb(255, 255, 255)
   if (colorString.toLowerCase().startsWith('rgb')) {
-    final List<int> rgb = colorString
+    final List<num> rgb = colorString
         .substring(colorString.indexOf('(') + 1, colorString.indexOf(')'))
         .split(',')
         .map((String rawColor) {
       rawColor = rawColor.trim();
       if (rawColor.endsWith('%')) {
         rawColor = rawColor.substring(0, rawColor.length - 1);
         return (parseDouble(rawColor) * 2.55).round();
+      } else if (rawColor.contains('.')) {
+        return parseDouble(rawColor);
       }
       return int.parse(rawColor);
```

Comment:
It would probably make sense to just have this be `return parseDouble(rawColor).round();` instead of checking for a `.`.